addons/website_sale_stock_wishlist/static/src/js/website_sale.js

```diff
@@ -0,0 +1,34 @@
+odoo.define("website_sale_stock_wishlist.website_sale", function (require) {
```

Comment:
Actually since this is a new file/widget, should probably use the new way of declaring those, see `/** @odoo-module **/`